src/NodeServices/WorkspaceEvents.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace DSNodeServices
```

Comment:
Perhaps it's just me, but I'd prefer to see a `Dynamo.Services` namespace where we put all the services from the future. For your case, it isn't even about *nodes* to begin with.